service/src/main/java/com/epam/rft/atsy/service/security/SpringSecurityAuthenticationService.java

```diff
@@ -32,7 +33,8 @@ public UserDetailsAdapter getCurrentUserDetails() throws UserNotLoggedInExceptio
   @Override
   public boolean isCurrentUserAuthenticated() {
     return SecurityContextHolder.getContext().getAuthentication() != null &&
-        SecurityContextHolder.getContext().getAuthentication().isAuthenticated();
+        SecurityContextHolder.getContext().getAuthentication().isAuthenticated() &&
+        !(SecurityContextHolder.getContext().getAuthentication() instanceof AnonymousAuthenticationToken);
```

Comment:
This new condition is not covered in the `SpringSecurityAuthenticationServiceTest`.